modules/ROOT/pages/auradb/change-data-capture/overview.adoc

```diff
@@ -0,0 +1,47 @@
+[[change-data-capture-overview]]
+= Change Data Capture (CDC)
+:description: This section covers changing transaction log enrichment mode using the Aura Console.
+
+label:AuraDB-Professional[] 
```

Comment:
CDC is not available in Aura Pro. There is a limited subset of customers who have Aura Enterprise databases, but use Aura Pro as a testing environment. For those, support may enable CDC on their Pro instances, if the customer asks for it.